### PR TITLE
entry updates

### PR DESCRIPTION
**Pull request by adri1mart1**
Hello
I have recently used your database and found it very useful. Some information needed to be updated, especially arround where I am located (Nantes). 
First commit is really about content updates.
The second commit is only a format suggestion as I noticed the france.json file was poorly formatted. 
Let me know if you want me to separate PRs and/or accept only the content commit.
Adrien M.

### CODE REVIEW

**Comment by adri1mart1** [comment]
@yoannsculo up :)

**Comment by yoannsculo** [comment]
Hello @adri1mart1,

Thank you very much for your advanced contribution ! This is much appreciated!
I am very sorry for this late answer. I wasn't aware of your proposal. My github account is linked to an email I no longer use.

Regarding your Pull Request, could you please add a changelog in your first commit with a summary of changes.
Because it is quite hard to follow changes in a json file. This is what I did to review your PR, but for the Git History it could be easier to follow changes with a small summary.

Here is an example with some changes I saw :
```
Add new companies / sites
- Add Kickmaker site of Nantes
- add Orthopus
- etc.

Update companies / sites
- Atmel renamed Microchip
- DCNS renamed into Naval Group
- Kiwatch moved
- Parkeon renamed Flowbird
- Flowbird Nantes moved
- etc.

Remove companies / sites
- remove Alcatel Orvault site
- remove a site from Intel
- etc.```

You can use one commit or split in 3, as you wish. Don't bother with several PRs.

I have comments for some entries (See my review)
Yoann

File: societes/france.json [hunk 1]
```diff
@@ -6243,6 +6289,54 @@
...
+            "locations": [
+                {
+                    "name": "",
+                    "postal_address": "52 Cours Jean Jaurès, 38000 Grenoble",
+                    "gps_coordinates": "45.18760/5.71854"
+                }
+            ]
+        },
+        {
+            "name": "MedTech spécialisée dans l'autotransfusion sanguine",
```
Comment (yoannsculo) [inline societes/france.json:5362]: Small mistake, description field in the name.

File: societes/france.json [hunk 2]
```diff
@@ -7951,6 +8050,8 @@
             "name": "Netatmo",
             "url": "http://www.netatmo.com",
             "description": "Smart home",
+            "url": "https://www.netatmo.com",
```
Comment (yoannsculo) [inline societes/france.json:6861]: this is a duplicate (see the lines above)

File: societes/france.json [hunk 3]
```diff
@@ -12378,6 +12457,38 @@
                 }
             ]
         },
+        {
+            "name": "Wiketio",
```
Comment (yoannsculo) [inline societes/france.json:10585]: Be careful, this is witekio and not wiketio.
It already exists (at the beginning of the json because it was Adeneo before). I know i need to do some cleanup to reorder all companies.

Comment (adri1mart1) [reply societes/france.json]: fixed

File: societes/france.json (hunk 2 above)
Comment (adri1mart1) [reply societes/france.json]: spotted ! fixed

File: societes/france.json (hunk 1 above)
Comment (adri1mart1) [reply societes/france.json]: fixed

File: societes/france.json [hunk 4]
```diff
@@ -6243,6 +6289,54 @@
...
+            "locations": [
+                {
+                    "name": "",
+                    "postal_address": "45 Rue Frédéric Joliot Curie, Marseille, France",
+                    "gps_coordinates": "43.3439196/5.4329399"
+                }
+            ]
+        },
+        {
+            "name": "Iskn",
```
[the remark this answers is not in the document]
Comment (adri1mart1) [reply societes/france.json:6293]: It definitely seems alive in 2022

File: societes/france.json [hunk 5]
```diff
@@ -6243,6 +6289,54 @@
...
+            "locations": [
+                {
+                    "name": "",
+                    "postal_address": "45 Rue Frédéric Joliot Curie, Marseille, France",
+                    "gps_coordinates": "43.3439196/5.4329399"
+                }
+            ]
+        },
+        {
+            "name": "Ironova",
```
[the remark this answers is not in the document]
Comment (adri1mart1) [reply societes/france.json]: Now i have 0, the website seems more of a tech blog than a company that create a product

File: societes/france.json [hunk 6]
```diff
@@ -5196,21 +5272,6 @@
                 }
             ]
         },
-        {
-            "name": "Goji",
```
[the remark this answers is not in the document]
Comment (adri1mart1) [reply societes/france.json:5200]: Apparently yes

File: societes/france.json [hunk 7]
```diff
@@ -2929,67 +2974,74 @@
...
                     "gps_coordinates": "47.75106/-3.35365"
                 },
                 {
-                    "name": "DCNS - Propulsion",
+                    "name": "Naval Group - Propulsion",
                     "postal_address": "Indret 44620 La Montagne",
                     "gps_coordinates": "47.19651/-1.68331"
                 },
                 {
-                    "name": "DCNS - Sirhena",
```
[the remark this answers is not in the document]
Comment (adri1mart1) [reply societes/france.json:2972]: It's not, I don't remember why i removed this but the location has changed (See [41b00eb](https://github.com/yoannsculo/emploi/pull/38/commits/41b00eb09afd7548d4fda4146f0b8bcc8d1b944c)

**Comment by adri1mart1** [comment]
Hello @yoannsculo 
Apologize accepted for your late response, please accept mine as well for this late PR update :o 

I've splited everything into multiple commits. No more commit about json format (I'll eventually do it in a separate PR).

Hope it fits this time
